luarules/gadgets/unit_collision_damage_behavior.lua

```diff
@@ -202,5 +210,10 @@ function gadget:GameFrame(frame)
 			unitInertiaCheckFlags[unitID] = nil
 		end
 	end
+
+	for unitID, bool in pairs(fallingKillQueue) do
```

Comment:
`bool` is not descriptive of what the value represents, and in this case the value is not needed at all,.
```suggestion
	for unitID, _ in pairs(fallingKillQueue) do
```